csvbase/models.py

```diff
@@ -81,7 +81,10 @@ class User(Base):
 
 class UserEmail(Base):
     __tablename__ = "user_emails"
-    __table_args__ = (METADATA_SCHEMA_TABLE_ARG,)
+    __table_args__ = (
+        CheckConstraint("email_address != ''", "email_address_not_blank"),
```

Comment:
Possibly this is better as `email_address ~ '@'` which checks that whatever is in there is at least a plausible email address.  That's the same check we use on the "frontend" as well (via HTML's `pattern` attribute)

Reply:
Done thanks. I have it checking for the same create_user and update_user in svc.py as well.